components/FeatureHighlight/src/private/MDCFeatureHighlightView+Private.m

```diff
@@ -108,6 +109,12 @@ - (instancetype)initWithFrame:(CGRect)frame {
     _displayMaskLayer = [[MDCFeatureHighlightLayer alloc] init];
     _displayMaskLayer.fillColor = [UIColor whiteColor].CGColor;
 
+    _accessibilityView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 1, 1)];
+    _accessibilityView.isAccessibilityElement = YES;
+    _accessibilityView.accessibilityTraits = UIAccessibilityTraitButton;
+    _accessibilityView.accessibilityLabel = @"Dismiss highlight button";
+    [self addSubview:_accessibilityView];
```

Comment:
Did you explore making `self` be the a11y element?

Reply:
I did, but with that it caused more complications as then its subviews won't be accessible, and then I would have to essentially change the hierarchy and add a parentView on top of self causing other complications with the controller searching for a MDCFeatureHighlightView as its base view for animation and such

Reply:
Gotcha; thank you for the context.